aria-practices.html

```diff
@@ -2835,6 +2835,7 @@ <h4>Keyboard Interaction</h4>
             </ol>
           </li>
           <li>If the tab list is horizontal, it does not listen for <kbd>Down Arrow</kbd> or <kbd>Up Arrow</kbd> so those keys can provide their normal browser scrolling functions even when focus is inside the tab list.</li>
+          <li>When the tabpanel does not contain any focusable elements or the first element with content is not focusable, the tabpanel should set <code>tabindex="0"</code> to include it in the tab sequence of the page.</li>
```

Comment:
> or the first element with content is not focusable

What does that mean? Could you please elaborate?

Reply:
There needs to be more clarity when `tabindex=0` is put on the element with the `tabpanel` role.  So the modified examples now show one example where the tabindex is on the tabpanel and one that it is not based on the requirements of the first element being focusable in the tabpanel.